frontend/src/pages/login.js

```diff
@@ -40,43 +36,24 @@ const Login = () => {
 
   const onSubmit = (event) => {
     event.preventDefault();
-    sessionStorage.setItem("Username", user.username)
-    if (0) {
-      console.log(0);
-    } else {
-      router.push('/home');
-    }
-  }
+    // checks with db that user exists
+    let username = user.username;
+    let password = user.password;
 
-  // const onSubmit = (event) => {
-  //   event.preventDefault();
-  //   // checks with db that user exists
-  //   fetch('/insert/api/here', {
-  //     method: 'POST',
-  //     body: JSON.stringify(user),
-  //     headers: {
-  //       'Content-Type': 'application/json',
-  //     },
-  //   })
-  //     .then((res) => {
-  //       if (!res.ok) {
-  //         throw res;
-  //       }
-  //       return res.json();
-  //     })
-  //     .then((json) => {
-  //       // puts user into the localstorage
-  //       localStorage.setItem('member', JSON.stringify(json));
-  //       // log out button become visible
-  //       // setVisible(true);
-  //       // goto to home page
-  //       // history.push('/');
-  //     })
-  //     .catch((error) => {
-  //       // displays error for incorrect username/password
-  //       setError(`${error.statusText} - wrong username or password`);
-  //     });
-  // };
+    fetch('http://localhost:5000/login', {
```

Comment:
If we're using `https` instead of `http` like here we can send the pw over unencrypted trusting in the https security. That's a server-side thing which I can do, so I'll take a look and see what it'd take.